manifests/database/postgresql.pp

```diff
@@ -84,6 +84,8 @@
   $read_database_password      = $puppetdb::params::read_database_password,
   $read_database_host          = $puppetdb::params::read_database_host
 ) inherits puppetdb::params {
+  $port = scanf($database_port, '%i')[0]
```

Comment:
I'm not sure if this brings any benefit. This casts it to an int in complicated?

Reply:
I agree. The type conversion, even if it was `Integer(...)` is fragile.
Made a note to address this a part of a comprehensive "type everything" PR.